src/mudlet.cpp

```diff
@@ -292,10 +292,7 @@ mudlet::mudlet()
     mpTabBar->setTabsClosable(true);
     mpTabBar->setAutoHide(true);
     connect(mpTabBar, &QTabBar::tabCloseRequested, this, &mudlet::slot_close_profile_requested);
-    // TODO: Do not enable this option currently - although the user can drag
```

Comment:
I remember vivid discussion in the original pr hence the temporary deactivation here. The pr number eludes me. May be worthwhile to review

Reply:
Seems to work OK in testing

Reply:
See new gif

Reply:
Please click multiplay button to see 2 or 3 profiles simultaneously, then move the headers around

Reply:
Good catch.

Reply:
:man_facepalming: Um, that was why I left the comment there...